Prometheus/Histogram.cs

```diff
@@ -124,6 +141,9 @@ public void Observe(double val, long count)
                     if (val <= _upperBounds[i])
                     {
                         _bucketCounts[i].Add(count);
+                        if (exemplar.Length > 0)
```

Comment:
I suggest null-checking exemplar, to avoid surprising callers. Here and elsewhere, so we treat an exemplar of null as equivalent to empty array, to be more tolerant of the form of input that is provided.

Perhaps just `exemplar ??= Array.Empty<...>()` at the start of the method or such?

Reply:
Rider suggested `if (exemplar is { Length: > 0 })` for a null + length check. That works (I think) ?